build_runner/lib/src/generate/create_merged_dir.dart

```diff
@@ -109,8 +128,7 @@ Future<bool> createMergedOutputDir(
         .map((p) => '$p:packages/$p/')
         .join('\r\n');
     for (var dir in rootDirs) {
-      var packagesAsset =
-          new AssetId(packageGraph.root.name, p.url.join(dir, '.packages'));
+      var packagesAsset = new AssetId(root, p.url.join(dir, '.packages'));
```

Comment:
will `root` be null here sometimes? Is that OK?

Reply:
I think maybe this is supposed to be `packageGraph.root.name`? and its the `dir` that we want to remove/change?